product_docs/docs/pem/9/registering_agent copy.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: "Registering a PEM agent"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/agent-user-guide/8.0/registering_agent.html"
+# This file is moved from pem_agent/02_registering_agent.mdx
+redirects:
+- /pem/latest/pem_agent/02_registering_agent/
+- /pem/latest/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/09_registering_a_pem_agent/ 
+---
+
+You must register each PEM agent installed on a separate host with the PEM server. (The PEM agent is different from the PEM server host.) The registration process provides the PEM server with the information it needs to communicate with the agent. The PEM agent graphical installer for Windows supports self-registration for the agent. On a Linux host, you must use the pemworker utility to register the agent.
+
+The PEM agent package installer places the PEM agent in the `/usr/edb/pem/agent/bin` directory. To register an agent, include the `--register-agent` keywords along with registration details when invoking the pemworker utility. Run the `pemworker` utility as root user.
```

Comment:
I think stating that pemworker needs to be run by the root user here is sufficient in terms of stem sentence mentions. I don't think we need to repeat it before the other code blocks where we are running pemworker.

Reply:
Done